tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -57,9 +55,17 @@ Verify Unauthorized User Is Not Able To Spawn Jupyter Notebook
     AdminGroups In OdhDashboardConfig CRD Should Be        rhods-users
     AllowedGroups In OdhDashboardConfig CRD Should Be      rhods-users
     Reload Page
-    Menu.Navigate To Page    Applications    Enabled
-    Run Keyword And Expect Error  *  Launch Jupyter From RHODS Dashboard Link
-    Wait Until Page Contains    Page Not Found   timeout=15s
+    ${version_check}=    Is RHODS Version Greater Or Equal Than    1.20.0
```

Comment:
depending on when this gets merged this could already be removed

Reply:
I cannot now that in advance